internal/jobservice/repository/postgres.go

```diff
@@ -330,3 +316,26 @@ func (s *JSRepoPostgres) GetSubscribedJobSets(ctx context.Context) ([]Subscribed
 	}
 	return tuples, nil
 }
+
+func (s *JSRepoPostgres) purgeExpired(ctx context.Context) {
+	jobsetsStmt := fmt.Sprintf(`DELETE FROM jobsets WHERE Timestamp < (extract(epoch from now()) - %d);`, s.jobServiceConfig.PurgeJobSetTime)
+	jobServiceStmt := fmt.Sprintf(`DELETE FROM jobservice WHERE Timestamp < (extract(epoch from now()) - %d);`, s.jobServiceConfig.PurgeJobSetTime)
+
+	ticker := time.NewTicker(time.Duration(s.jobServiceConfig.PurgeJobSetTime) * time.Second)
+	log := log.WithField("JobService", "ExpiredJobSetsPurge")
+
+	for range ticker.C {
```

Comment:
It would be useful to explicitly log that the purge is starting once the loop starts. This way, when parsing through the logs in production, you'll be able to see when each purge cycle runs.